packages/cli/src/workflows/workflow.service.ee.ts

```diff
@@ -80,7 +80,10 @@ export class EnterpriseWorkflowService {
 		currentUser: User,
 	): Promise<void> {
 		workflow.usedCredentials = [];
-		const userCredentials = await this.credentialsService.getMany(currentUser, { onlyOwn: true });
```

Comment:
We don't seem to use `onlyOwn` anymore, perhaps this is always covered by roles now - let's remove if so?